src/Components/Feed/Posts.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react'
+import { Post } from './Post.jsx'
+import axios from 'axios';
+import { WithPost } from '../../HOCs/WithPost.jsx';
+import { useDoLike } from '../apis/useDoLike.jsx';
+import { useGetPost } from '../apis/useGetPost.jsx';
+import { useDoUnLike } from '../apis/useDoUnlike.jsx';
+import { PostLoader } from '../Loaders/PostLoader.jsx';
+import { WithToken } from "../../HOCs/withToken";
+import { useGetUser } from '../apis/useGetUser.jsx';
+import { useDeletePost } from '../apis/useDeletePost.jsx';
+
+
+export const PostsComponent = ({ statePost, setStatePost, user, setUser }) => {
+  const [postRes, getPost] = useGetPost();
+  const [res, doLike] = useDoLike()
+  const [unlikeRes, doUnLike] = useDoUnLike()
+  const [currentPostId, setCurrentPostId] = useState()
+  const [errorMessage, setErrorMessage] = useState('');
+  const [userRes, getUser] = useGetUser()
+  const[resDel,doDel]=useDeletePost()
+
+  useEffect(() => {
+    getUser()
+  }, [])
+
+  useEffect(() => {
+    if (userRes) {
```

Comment:
same. userRes will always be truthy as you defined initial value. what's the purpose of if here?

Reply:
this isn't resolved yet.